src/components/ScaleBar.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { ScaleBarIcon } from './Icons';
+
+import '../../css/scale_bar.css';
```

Comment:
This is making me remember that conversation we were having about styled-components. I can definitely see the use case here!

Reply:
Yeah feeling some time pressure to get MVP done with the way I'm used to developing, but want to move into that CSS-in-JS approach soon, maybe if there is a conversation around bumping the antd version in sim-website at some point.